pkg/controller/iampolicy/iampolicy_controller.go

```diff
@@ -301,12 +301,12 @@ func convertMaptoPolicyNameKey() map[string]*policiesv1.IamPolicy {
 	return plcMap
 }
 
-func addViolationCount(plc *policiesv1.IamPolicy, userCount int, namespace string) bool {
+func addViolationCount(plc *policiesv1.IamPolicy, roleName string, userCount int, namespace string) bool {
 
 	changed := false
 	// DO NOT change the message below without also considering that it is parsed to obtain
 	// the count from the previous status!
-	msg := fmt.Sprintf("Number of users with clusteradmin role is %s above the specified limit", fmt.Sprint(userCount))
+	msg := fmt.Sprintf("The number of users with the %s role is %d above the specified limit", roleName, userCount)
```

Comment:
good that the specific cluster role was added to the msg , but could we keep userCount part as is ( %s with fmt.Sprint(userCount) )
especially given the note above the msg, not sure exactly but  this is probably how the governance framework etc that parses the msg on the hub expects it.

Also, the associated e2e test in governance-framework may also need to fixed to expect this msg,  to avoid a e2e test failure